src/utils/dateUtils.ts

```diff
@@ -20,7 +20,7 @@ export function formatterDatoTid(dato: string | Date) {
     return moment(dato).format(DATO_TID_FORMAT);
 }
 
-export function formatterDatoTidMedMaanedsnavn(dato: string | Date) {
+export function formatterDatoTidMedMaanedsnavn(dato?: string | Date) {
```

Comment:
Hvorfor er denne gjort optional?

Reply:
For å slippe å måtte gjøre undefined-sjekken overalt denne brukes, bedre å sentralisere. Skaper dette problemer?

Reply:
aha, nope, bare så ikke at du hadde tatt den i bruk på den måten noe sted, men burde være helt fint det